docs/getting_started/index.rst

```diff
@@ -1,115 +1,164 @@
 Getting Started
 ===============
 
-.. note:: This document assumes you have a `working Ansible
+.. Note:: This document assumes you have a `working Ansible
           installation`_. If you don't, install Ansible before
           continuing. This can be done simply by running ``pip install -r
           requirements.txt`` from the root of the project.
 
-The Mantl project uses Ansible to bring up nodes and clusters. This generally
-means that you need three things:
+          It also assumes you have a working Terraform installation which you
+          can download from `Terraform downloads`_.
 
- 1. hosts to use as the base for your cluster
- 2. an `inventory file`_ with the hosts you want to be modified.
- 3. a playbook to show which components should go where.
+General Information about Mantl with Ansible
+============================================
 
-Provisioning Cloud Hosts
-------------------------
+The Mantl project uses Ansible to bring up
+nodes and clusters. This generally means that you need three things:
+
+1. Hosts to use as the base for your cluster
+2. An `inventory file`_ with the hosts you want to be modified. Mantl includes
+   an Ansible inventory for terraformed hosts, `terraform.py`_.
+3. A playbook matching Ansible roles to hosts. Mantl organizes its components
+   in `sample.yml`_, which we recommend copying to ``mantl.yml`` for the
+   possibility of later customization. You can read more about `playbooks`_ in
+   the `Ansible docs`_.
+
+Preparing to provision Cloud Hosts
+>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
```

Comment:
Looks like `#`, `*`, `=`, `-`, `^`, `"` officially. So we're already off... But let's maybe use `^`, yes.